### ISSUE TITLE
Xenochimera feral proc for viewers is borked

### ISSUE THREAD

**Issue by Screemonster** [VOREStation/VOREStation#2518]
#### Brief description of the issue
Xenochimera are supposed to not get spam messages or hallucinations when they have company, in order to not disrupt scenes. In fact, they do.

#### What you expected to happen
Proc to work

#### What actually happened
Doesn't.

#### Anything else you may wish to add:
- It's supposed to be handled in this line, but there's no break in the first bit if it finds a mob other than yourself:
https://github.com/VOREStation/VOREStation/blob/d6767e9560aee3e11955de5e11670443f89a3c5b/code/modules/mob/living/carbon/human/species/station/station_special_vr.dm#L125

It also doesn't account for finding your mob first and hallucinatifying you anyway regardless of whether there are more people about.

tl;dr that proc needs a better way of checking "do I have company" but it's like half ten at night and I've got work in the morning and I'm not wrestling with byond to try and bash out a fix now. Creating issue to remind me later.
